__tests__/app.test.js

```diff
@@ -1,51 +1,88 @@
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+const connection = require("../db/connection");
+const data = require("../db/data/test-data");
+const description = require("../endpoints.json");
 
-afterAll(()=>{
-    return connection.end()
-})
+afterAll(() => {
+  return connection.end();
+});
 
-beforeEach(()=>{
-    return seed(data)
-})
+beforeEach(() => {
+  return seed(data);
+});
 
-describe('/api/topics', () => {
-  test('GET: 200 - should respond with status 200 and an array of topic objects', () => {
+describe("/api/topics", () => {
+  test("GET: 200 - should respond with status 200 and an array of topic objects", () => {
     return request(app)
-    .get('/api/topics')
-    .expect(200)
-    .then(({body}) => {
-        const topics = body.topics.rows
-        expect(topics.length).toBe(3)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const topics = body.topics.rows;
+        expect(topics.length).toBe(3);
         topics.forEach((topic) => {
-            expect(typeof topic.slug).toBe('string')
-            expect(typeof topic.description).toBe('string')
-        })
-    })
-  })
-  test('GET: 404 - should respond with an error message saying not found when we try to access a non-existent url', () => {
+          expect(typeof topic.slug).toBe("string");
+          expect(typeof topic.description).toBe("string");
+        });
+      });
+  });
+  test("GET: 404 - should respond with an error message saying not found when we try to access a non-existent url", () => {
+    return request(app)
+      .get("/api/wrong_url")
+      .expect(404)
+      .then((body) => {
+        expect(body.statusCode).toBe(404);
+        expect(body.res.statusMessage).toBe("Not Found");
+      });
+  });
+});
+
+describe("/api", () => {
+  test("GET: 200 - should provide a description of API endpoints available", () => {
     return request(app)
-    .get('/api/wrong_url')
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.description).toEqual(description);
+      });
+  });
+});
+
+describe("/api/articles/:article_id", () => {
+  test("GET: 200 - should return an article by its id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article;
+        expect(article.author).toBe("butter_bridge");
+        expect(article.title).toBe("Living in the shadow of a great man");
+        expect(article.article_id).toBe(1);
+        expect(article.body).toBe("I find this existence challenging");
+        expect(article.topic).toBe("mitch");
+        expect(article.created_at).toBe("2020-07-09T20:11:00.000Z");
+        expect(article.votes).toBe(100);
+        expect(article.article_img_url).toBe(
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        );
+      });
+  });
```

Comment:
Great that you have made this test flexible by not checking it equals exactly another object, but rather that it contains the right info!
You can also achieve this using matchers like `to MatchObject`